### PR TITLE
Add Extra Resources

### PR DESCRIPTION
**Pull request by khairahscorner**
## Description

Added extra resources to the Resources page
- Link to video on how to use mentorship notion template
- Link to PDF on how to set up calendly

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [x] Other

## Screenshots

**After**
Screenshot 2024-02-08 at 1 17 15 pm

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/resources.yml [hunk 1]
```diff
@@ -10,3 +10,9 @@
 - title: Mentor's Checklist
   link: https://github.com/WomenWhoCode/WWCode-London/blob/main/resources/mentorship/images/checklist-mentor.png
   image: assets/images/resources/checklist-mentors.png
+- title: Notion Template Guide
```
Comment (dricazenck) [inline _data/resources.yml:13]: What about "How to use mentoring template?" or "How to use Notion template?"

**Comment by dricazenck** [comment]
I liked a lot the images, 🥰

File: _data/resources.yml (hunk 1 above)
Comment (khairahscorner) [reply _data/resources.yml:13]: Yeah, that was my first thought but it broke to a second line so the alignment was off
Screenshot 2024-02-08 at 11 53 47 pm

Comment (dricazenck) [reply _data/resources.yml:13]: @nora-weisser you are good with names, what do you think?

Comment (dricazenck) [reply _data/resources.yml:13]: @khairahscorner could you try to reduce the space between resources and/ or increase a bit the box? 

Other suggestions:
- Mentorship Template 
- Sessions Tracking

**Review by dricazenck — APPROVED** [review]
For my point of view it is good. In case you want to try the other names or keep this one it is also fine by me. Thank you a lot ❤️

File: _data/resources.yml (hunk 1 above)
Comment (nora-weisser) [reply _data/resources.yml:13]: Another suggestion: `Progress Tracking` or `Progress Tracking Template` (if it fits) :)

Comment (khairahscorner) [reply _data/resources.yml]: Progress Tracking template sounds good @nora-weisser. I've also reduced the card gap, so the card is a bit bigger, and I updated the name for Calendly to be more descriptive too, what do we think @dricazenck?

Screenshot 2024-02-11 at 3 56 38 am

**Pull request merged**